tools/shared/src/main/scala/org/scalajs/core/tools/linker/backend/emitter/ScalaJSClassEmitter.scala

```diff
@@ -988,6 +990,16 @@ private[emitter] final class ScalaJSClassEmitter(semantics: Semantics,
     genClassOrModuleExportDef(cd, tree.fullName, baseAccessor)
   }
 
+  def genTopLevelModuleExportDef(cd: LinkedClass,
```

Comment:
Same here about the "lazyness". It is not clear from the names what should be different between `genTopLevelModuleExportDef` and `genModuleExportDef`.